atom/browser/browser.h

```diff
@@ -156,7 +156,7 @@ class Browser : public WindowListObserver {
   };
 
   // Add a custom task to jump list.
-  void SetUserTasks(const std::vector<UserTask>& tasks);
+  bool SetUserTasks(const std::vector<UserTask>& tasks);
```

Comment:
Can you also update the docs to mention this method now returns whether it was successful? Thanks.

https://github.com/electron/electron/blob/master/docs/api/app.md#appsetusertaskstasks-windows